src/main/cs/Infinispan/HotRod/Config/ServerConfigurationBuilder.cs

```diff
@@ -4,32 +4,22 @@
 namespace Infinispan.HotRod.Config
 {
 #pragma warning disable 1591
-    public class ServerConfigurationBuilder : AbstractConfigurationChildBuilder, IBuilder<ServerConfiguration>
+    public class ServerConfigurationBuilder 
     {
         private Infinispan.HotRod.SWIG.ServerConfigurationBuilder builder;
-
+        private ConfigurationBuilder parent;
         internal ServerConfigurationBuilder(ConfigurationBuilder parent,
-                                            Infinispan.HotRod.SWIG.ServerConfigurationBuilder builder) : base(parent)
+                                            Infinispan.HotRod.SWIG.ServerConfigurationBuilder builder)
         {
             this.builder = builder;
-        }
-
-        public void Validate()
-        {
-            builder.validate();
+            this.parent = parent;
```

Comment:
Is the parent required to be stored? The next class in this PR (SSLConfigurationBuilder) doesn't store it.